js/binance.js

```diff
@@ -4934,4 +4935,53 @@ module.exports = class binance extends Exchange {
             'info': response,
         };
     }
+
+    async fetchBorrowRateHistory (code, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (limit > 93) {
```

Comment:
We should not treat the `limit` here as the number of days. Limit means `how many entries to return`. I suggest that we remove all the "93" arithmetics from this method and omit the limit if it's undefined.

Reply:
There is one entry per day, so that's why it's number of days

Reply:
> We should not treat the `limit` here as the number of days. Limit means _how many entries to return_, not _how many days_. I suggest that we remove all the "93" arithmetics from this method and omit the limit if it's undefined or subtract 3 months like `3 * 31 * 24 * 24 * 60 * 1000` or `3 * 30 * 24 * 60 * 60 * 1000`. Alternatively, i would probably write some logic for adding and subtracting calendar dates. Will get to this PR shortly.

I had written this logic on a previous commit that got reverted, it's not 3 months, it's just a limit of 93, and if you're over 93, you get weird errors that don't make any sense

Reply:
If the `limit` is undefined, then `if (limit > 93)` will crash Python, we should rewrite this to:

```JavaScript
if (limit === undefined) {
    limit = 93;
} else {
    limit = Math.min (limit, 93);
}
```